tests/integration/go_ethereum/common.py

```diff
@@ -8,7 +8,8 @@
     Web3ModuleTest,
 )
 
-GETH_SIGNED_TX = b'\xf8j\x80\x85\x040\xe24\x00\x82R\x08\x94\xdcTM\x1a\xa8\x8f\xf8\xbb\xd2\xf2\xae\xc7T\xb1\xf1\xe9\x9e\x18\x12\xfd\x01\x80\x86\xee\xca\xc4f\xe1\x16\xa0\xbb7^\x1f\xf0\x03(P\x07|\x053Q\xd3M\xf1\x83\xe9\xdcp\xdc\x02\xb4\xe7`\x85\xcd\x84\xdb\xb4\xd0\xaa\xa07\x8cl\xd7\xa6R\x01\xfaW\x0e\x0f\xc1_$\xdf`\x8dO\x18\x1dC\xbc\x87\x8fud\xd2R*W\xfd4'  # noqa: E501
+GETH_17_SIGNED_TX = b'\xf8j\x80\x85\x040\xe24\x00\x82R\x08\x94\xdcTM\x1a\xa8\x8f\xf8\xbb\xd2\xf2\xae\xc7T\xb1\xf1\xe9\x9e\x18\x12\xfd\x01\x80\x86\xee\xca\xc4f\xe1\x16\xa0\xbb7^\x1f\xf0\x03(P\x07|\x053Q\xd3M\xf1\x83\xe9\xdcp\xdc\x02\xb4\xe7`\x85\xcd\x84\xdb\xb4\xd0\xaa\xa07\x8cl\xd7\xa6R\x01\xfaW\x0e\x0f\xc1_$\xdf`\x8dO\x18\x1dC\xbc\x87\x8fud\xd2R*W\xfd4'  # noqa: E501
+GETH_18_SIGNED_TX = b'\xf8i\x80\x84;\x9a\xca\x00\x82R\x08\x94\xdcTM\x1a\xa8\x8f\xf8\xbb\xd2\xf2\xae\xc7T\xb1\xf1\xe9\x9e\x18\x12\xfd\x01\x80\x86\xee\xca\xc4f\xe1\x16\xa0W\x91\xe6a\xb7l\x93,\xd3\xdd?;\xa8\x0e\xd3\xbe\x04\x89(\xd49\x1e+\xd6\xee\x88k\xfb\xe7\x83\xeb(\xa0\x04\x9a%`\x91uc\x1a\xdd\xb8\x96\xbb\x10\xb5v|\xcf\x03\xb6\x90\xbb\x93x\xfef\xae\xe1L\xbc7\xafJ'  # noqa: E501
 
```

Comment:
I'm having a hard time figuring out why these are returning different values for a signed tx. If the parameters are the same (i.e. gas price, to / from, tx...) shouldn't the signed transaction be the same? Why could different versions of geth be returning different values?

Reply:
I'd decode it (`rlp.decode(...)`) and see what is different.